### PR TITLE
doc/cephfs: improve add/remove MDS section

### PR DESCRIPTION
**Pull request by batrick**
Include hardware details and update language for modern tools.

Fixes: http://tracker.ceph.com/issues/39620
Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

### CODE REVIEW

**Comment by joscollin** [comment]
jenkins render docs

File: doc/cephfs/add-remove-mds.rst [hunk 1]
```diff
@@ -1,51 +1,105 @@
...
+	$ sudo systemctl stop mds.${id}
 
-#. Stop the old Metadata Server and start using the new one. ::
+The MDS will automatically notify the Ceph monitors that it is going down. This
+enables the monitors to perform instantaneous failover to an available standby,
+if one exists. It is unnecessary to use administrative commands to effect this
+failover, e.g. through the use of ``ceph mds fail mds.${id}``.
 
-	$ ceph mds fail <mds name>
+#. Remove the ``/var/lib/ceph/mds/ceph-${id}`` directory on the MDS. ::
```
Comment (joscollin) [inline doc/cephfs/add-remove-mds.rst]: Wrong paragraph number `1`

**Comment by batrick** [comment]
jenkins render docs

**Comment by batrick** [comment]
jenkins render docs

**Comment by batrick** [comment]
jenkins render docs

**Comment by batrick** [comment]
jenkins render docs

**Pull request merged**